src/main/java/Duke/task/Task.java

```diff
@@ -0,0 +1,64 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents a task.
+ */
+abstract public class Task {
+
+    private String details;
+    private boolean completed;
```

Comment:
Maybe this boolean variable can be named `isCompleted` as well?
```suggestion
    private boolean isCompleted;
```